packages/api/src/schema/schema.yaml

```diff
@@ -1211,6 +1211,11 @@ components:
           type: string
           readOnly: true
           description: URL to access file via HTTP through an IPFS gateway
+        createdAt:
```

Comment:
I think this field should be named `updatedAt` instead, since the `createdAt` field in the API schema right now is immutable, meaning when the object was originally created, not when they changed the last time.

In this case, the IPFS export especially of the `nftMetadata` can change multiple times. The "original file" export can also be changed, at least in terms of unpinning (`ipfs: false`) and re-pinning (`ipfs: true` again) it. 

So IMO it'd make more sense for this to be `updatedAt` instead. WDYT?

Reply:
It makes sense. I used the `createdAt` because that was what I found in the task. But it makes sense to use `updatedAt` and generate it.